### PR TITLE
Update User Guide for listOrder Command and FooChao.md

### LINKED ISSUE #65: Update User Guide for listOrder Command and FooChao.md

### PR DESCRIPTION
**Pull request by FooChao**
close #65

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -194,12 +208,13 @@ _Details coming soon ..._
...
-**List**   | `list`
-**Help**   | `help`
+Action                | Format, Examples
+----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**Add person**        | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Clear**             | `clear`
+**Delete person**     | `delete INDEX`<br> e.g., `delete 3`
+**Edit person**       | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
+**Find person**       | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
+**List persons**      | `list`
```
Comment (FooChao) [inline docs/UserGuide.md:218]: persons is used in javadocs but idk if it is a typo

Comment (kaixuan477) [reply docs/UserGuide.md:218]: Another alternative is to state "List all persons"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -194,12 +208,13 @@ _Details coming soon ..._
...
-**Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear**  | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List**   | `list`
-**Help**   | `help`
+Action                | Format, Examples
+----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**Add person**        | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
```
[the remark this answers is not in the document]
Comment (kaixuan477) [reply docs/UserGuide.md:213]: Similar comments on adding "a" person for specificity of command, albeit supposedly inferred from singular "person"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -145,6 +145,20 @@ Examples:
...
+
+Format: `listOrder INDEX`
+
+* List all orders under the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `listOrder 2` lists all orders under the 2nd person in the address book.
+* `find Betsy` followed by `listOrder 1` lists all order under the 1st person in the results of the `find` command.
```
Comment (kaixuan477) [inline docs/UserGuide.md:160]:
```suggestion
* `find Betsy` followed by `listOrder 1` lists all orders of the 1st person from the list of persons found with "Betsy".
```

**Review by kaixuan477 — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (FooChao) [reply docs/UserGuide.md]: ok

**Pull request merged**